Objects/typevarobject.c

```diff
@@ -443,45 +443,39 @@ static PyMethodDef typevar_methods[] = {
 PyDoc_STRVAR(typevar_doc,
 "Type variable.\n\
 \n\
-Usage::\n\
+The preferred way to construct a type variable is via the dedicated syntax\n\
+for generic functions, classes, and type aliases:\n\
 \n\
-  T = TypeVar('T')  # Can be anything\n\
-  A = TypeVar('A', str, bytes)  # Must be str or bytes\n\
+   class Sequence[T]:  # T is a TypeVar\n\
+         ...\n\
```

Comment:
There's some weird indentation here:

```suggestion
       ...\n\
```

Reply:
Thanks, made these all consistently use four-space indents.